ex2_hal/iris/hardware_interface/source/iris.c

```diff
@@ -15,10 +15,13 @@
 #include "FreeRTOS.h"
 #include <string.h>
 #include <stdlib.h>
+#include <stdio.h>
```

Comment:
Does this use stdio? Should remove this include if it doesn't

Reply:
Removed :+1: